src/main/java/com/exadel/sandbox/mapper/impl/EmployeeMapperImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.exadel.sandbox.mapper.impl;
+
+import com.exadel.sandbox.dto.EmployeeDto;
+import com.exadel.sandbox.entities.Employee;
+import com.exadel.sandbox.mapper.EmployeeMapper;
+import lombok.RequiredArgsConstructor;
+
+
+@RequiredArgsConstructor
+public class EmployeeMapperImpl implements EmployeeMapper {
+
+    @Override
+    public Employee toEntity(EmployeeDto employeeDto) {
+        return new Employee(
```

Comment:
Please use model mapper for these purposes https://www.baeldung.com/java-modelmapper
Please change other mappers